src/AboutDialog.cpp

```diff
@@ -348,39 +348,21 @@ Audacity is [[https://www.audacityteam.org/download|available]] for Windows, Mac
 /* Do the i18n of a string with markup carefully with hints.
  (Remember languages with cases.) */
       XO(
-/* i18n-hint: First and third %s will be the program's name,
-  second %s will be "volunteers", fourth "available" */
-"%s is a free program written by a worldwide team of %s. \
-%s is %s for Windows, Mac, and GNU/Linux (and other Unix-like systems).")
+/* i18n-hint: %s substitutes to the program name. Do not change the links (ie the left side of the [[ | ]] ) */
+"%s is free, open source software, developed by [[https://mu.se|Muse Group]] with support of the [[https://www.audacityteam.org/about/credits|Free/Open Source community]].  \
+%s is [[https://www.audacityteam.org/download|available]] for Windows, MacOS and GNU/Linux.")
```

Comment:
Do not remove "excessive" substitutions.

I wrote those complications in for a very good reason:  we don't want confusing markup to be in the translation catalog, and prone to errors made by translators.

Therefore the complication of substituting a translated word into a marked-up link, and then substitution of that into a bigger sentence.